addons/binding/org.openhab.binding.yeelight/src/main/java/org/openhab/binding/yeelight/internal/YeelightBindingConstants.java

```diff
@@ -36,6 +37,7 @@
 
     // List of thing Parameters names
     public static final String PARAMETER_DEVICE_ID = "deviceId";
+    public static final String PARAMETER_Duration = "duration";
```

Comment:
Afaik, final variable names should be all uppercase.

Reply:
You are right, I will fix that.